### PR TITLE
feat(SWR): add swr image auto sync resource

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  add swr image auto sync resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
add swr image auto sync resource
```

## PR Checklist

* [ ] Tests added/passed.
* [ ] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST=
'./huaweicloud/services/acceptance/swr/' TESTARGS='-run TestAccSwrImageSync_basic'
==> Checking that code complies with gofmt requirements... 
TF_ACC=1 go test ./huaweicloud/services/acceptance/swr/ -v -run TestAccSwrImageSync_basic -timeout 360m -parallel 4 
=== RUN   TestAccSwrImageSync_basic 
=== PAUSE TestAccSwrImageSync_basic 
=== CONT  TestAccSwrImageSync_basic 
--- PASS: TestAccSwrImageSync_basic (11.20s) 
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/swr       11.254s

```

### CODE REVIEW

**Comment by houpeng80** [comment]
make testacc TEST='./huaweicloud/services/acceptance/swr/' TESTARGS='-run TestAccSwrImageSync_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/swr/ -v -run TestAccSwrImageSync_basic -timeout 360m -parallel 4
=== RUN   TestAccSwrImageSync_basic
=== PAUSE TestAccSwrImageSync_basic
=== CONT  TestAccSwrImageSync_basic
--- PASS: TestAccSwrImageSync_basic (14.86s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/swr       14.936s

File: docs/resources/swr_image_auto_sync.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* `updated_at` - Indicates the update time.
+
+## Import
+
+The SWR image auto sync can be imported using the organization name, repository name,
+target region and target organization separated by a slash, e.g.:
+
+```bash
+$ terraform import huaweicloud_swr_image_auto_sync.test 
+$ <organization_name>/<repository_name>/<target_region>/<target_organization>
```
Comment (deer-hang) [inline docs/resources/swr_image_auto_sync.md]: how about make it one line

Comment (houpeng80) [reply docs/resources/swr_image_auto_sync.md]: done

**Comment by houpeng80** [comment]
make testacc TEST='./huaweicloud/services/acceptance/swr/' TESTARGS='-run TestAccSwrImageAutoSync_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/swr/ -v -run TestAccSwrImageAutoSync_basic -timeout 360m -parallel 4
=== RUN   TestAccSwrImageAutoSync_basic
=== PAUSE TestAccSwrImageAutoSync_basic
=== CONT  TestAccSwrImageAutoSync_basic
--- PASS: TestAccSwrImageAutoSync_basic (28.15s) 
PASS 
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/swr       28.197s

**Pull request merged**